bin/anycablebility

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require 'bundler/setup'
+require 'docopt'
```

Comment:
Why did you choose `docopt`? We can use [OptionParse](http://ruby-doc.org/stdlib-2.4.0/libdoc/optparse/rdoc/OptionParser.html), which is a common practice. Also we can avoid the additional dependency.